### PR TITLE
ensure show does not exist before adding for untyping

### PR DESCRIPTION
**Pull request by mshick**
#3

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -51,7 +51,9 @@ module.exports = {
 
 					// give 'show' class to all children immediately
 					for (var j = 0; j < children.length; j++) {
-						children[j].className += ' show';
+						if (children[j].className.search('show') < 0) {
```
Comment (gabrielflorit) [inline index.js:54]: Is `search` JS? or jQuery? Hadn't seen this method before.

Comment (mshick) [inline index.js:54]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/search since ES3 ;)

Comment (gabrielflorit) [inline index.js:54]: Huh! Fantastic, learn something new every day.

**Comment by gabrielflorit** [comment]
Thanks for noting this, lazy bug on my part. I decided to use [element-class](https://github.com/maxogden/element-class) to handle `className` manipulation.

**Pull request closed without merging**